index.html

```diff
@@ -4,9 +4,152 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Youtube page by Adorer</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-    
+    <nav class = "flex-div">
+        <div class="nav-left flex-div">
+            <img src="images/menu.png" alt="menu-icon" class="menu-icon">
+            <img src="images/logo.png" alt="youtube icon" class="logo-icon">
+        </div>
+
+        <div class="nav-center flex-div">
+            <div class="search-box flex-div">
+                <input type="text" placeholder="Search">
+                <img src="images/search.png" alt="search-icon">
+             </div>
+             <img src="images/voice-search.png" alt="voice-icon" class = "voice-icon">
+        </div>
+
+        <div class="nav-right flex-div" >
+            <img src="images/upload.png" alt="upload-icon">
+            <img src="images/more.png" alt="images-icon">
+            <img src="images/notification.png" alt="notification-icon">
+            <img src="images/Jack.png" alt="menu icon" class = "user-icon">
+        </div>
```

Comment:
- [ ] **[OPTIONAL]:** Great job implementing the suggestion by the previous reviewer! 💯 👏🏼 It would be great if you can give a more detailed explanation of the requested change. Usually, when you ask to change HTML elements is because you can improve the semantics or the accessibility of your page! 💻 You could rephrase your comment like this, for example:

>_I think it will be better to use an unordered list (ul and li) here and each list item should be wrapped in an anchor tag. This change will improve the semantics of the page since it's representing a list of interactive icons and could help the browsers and assistive tools such as screen readers to have a better time navigating through your page._

It would be great if you change this! 🤓 🛠️